distribution/server-quarkus/src/main/README.txt

```diff
@@ -0,0 +1,12 @@
+Keycloak server powered by Quarkus.
+
+To run on Linux/Unix:
+
+$ cd bin
+$ ./kc.sh
+
+To run on Windows:
+
+> ...\bin\kc.bat
+
+After the server boots, open http://localhost:8080/auth in your web browser. The welcome page will indicate that the server is running.
```

Comment:
"/auth" was removed in Quarkus dist. Change to "http://localhost:8080"

Reply:
Changed as requested